Source/WebCore/css/StyleProperties.h

```diff
@@ -60,9 +60,10 @@ class StyleProperties : public RefCounted<StyleProperties> {
         const CSSValue* value() const { return m_value; }
         // FIXME: We should try to remove this mutable overload.
         CSSValue* value() { return const_cast<CSSValue*>(m_value); }
+        Ref<CSSValue> protectedValue() const { return const_cast<CSSValue&>(*m_value); }
 
         // FIXME: Remove this.
-        CSSProperty toCSSProperty() const { return CSSProperty(id(), const_cast<CSSValue*>(m_value), isImportant() ? IsImportant::Yes : IsImportant::No, m_metadata.m_isSetFromShorthand, m_metadata.m_indexInShorthandsVector, isImplicit()); }
+        CSSProperty toCSSProperty() const { return CSSProperty(id(), protectedValue(), isImportant() ? IsImportant::Yes : IsImportant::No, m_metadata.m_isSetFromShorthand, m_metadata.m_indexInShorthandsVector, isImplicit()); }
```

Comment:
Why `protectedValue` here instead of `*value()`? I heard @cdumez say we try not to use those functions like this, but maybe we'd have to write `Ref` here otherwise and also I might be misremembering his remark/guideline?

Or maybe this is about our mixed up use of `const` here?

Seems like we should change `value()` to return a reference in a follow up.